src/Tribe/Support.php

```diff
@@ -417,13 +417,13 @@ public static function ajax_sysinfo_optin() {
 		public static function send_sysinfo_key( $optin_key = null, $url = null, $remove = null, $pueadd = false ) {
 			$url = $url ? $url : urlencode( str_replace( [ 'http://', 'https://' ], '', get_site_url() ) );
 
-			$teccom_url = 'https://theeventscalendar.com/';
+			$teccom_url = 'https://support-api.tri.be';
```

Comment:
I would rather not change this in the plugin as the URL isn't going to be tri.be for ever and we do not have control of rewrite rules for tri.be domains. I'll work on getting the theeventscalendar.com url to redirect.

Reply:
@borkweb - I totally understand that logic, should I close this PR then or just revert the URL change?

Reply:
I've reverted the URL change